### DISCUSSION TITLE
Add umask calculator to existing chmod calculator

### DISCUSSION THREAD

**Discussion by ovizii** [CorentinTh/it-tools#719] [Ideas]
Something like this: https://wintelguy.com/umask-calc.pl

**Comment by sharevb** [comment]
Hi @ovizii and @CorentinTh , implemented in #964

**Comment by sharevb** [comment]
Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
